OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -37,20 +37,12 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Should the projectile's shadow be rendered?")]
 		public readonly bool Shadow = false;
 
-		[Desc("Minimum vertical launch angle (pitch).")]
-		public readonly WAngle MinimumLaunchAngle = new WAngle(-64);
+		[Desc("Vertical launch angle (pitch). If two values are defined, they represent minimum and maximum.")]
+		public readonly WAngle[] LaunchAngle = { new WAngle(-64), new WAngle(128) };
 
-		[Desc("Maximum vertical launch angle (pitch).")]
-		public readonly WAngle MaximumLaunchAngle = new WAngle(128);
-
-		[Desc("Minimum launch speed in WDist / tick. Defaults to Speed if -1.")]
-		public readonly WDist MinimumLaunchSpeed = new WDist(-1);
-
-		[Desc("Maximum launch speed in WDist / tick. Defaults to Speed if -1.")]
-		public readonly WDist MaximumLaunchSpeed = new WDist(-1);
-
-		[Desc("Maximum projectile speed in WDist / tick")]
-		public readonly WDist Speed = new WDist(384);
+		[Desc("Projectile speed in WDist / tick, two values indicate launch speed and maximum speed,",
+			"three values represent minimum launch speed, maximum launch speed and maximum speed (in that order).")]
+		public readonly WDist[] Speed = { new WDist(384) };
```

Comment:
Notcertain if I'm a fan of this, this forces unneeded duplication at inherited ones.